netkan/netkan/scheduler.py

```diff
@@ -42,7 +43,8 @@ def _in_group(self, netkan):
             return self.nonhooks_group
 
     def schedule_all_netkans(self):
-        messages = (nk.sqs_message() for nk in self.netkans() if self._in_group(nk))
+        messages = (nk.sqs_message(CkanGroup(self.ckan_meta_path, nk.identifier))
+                    for nk in self.netkans() if self._in_group(nk))
         for batch in sqs_batch_entries(messages):
```

Comment:
I don't think it's a big deal, but do we know what the performance hit of scheduling like this?

Reply:
I couldn't think of a good way to measure that. Ballparking, it adds disk reads and JSON parsing, to the tune of several times what's currently done (since each module has one .netkan and multiple .ckans). The scheduler might take, say, 10x as much time to run, but I'm pretty sure it's not the current bottleneck, either.

Reply:
Admittedly my dev machine at work is pretty beefy, single threaded performance isn't going to be too different
```python
In [18]: from netkan.metadata import Ckan                                                                                                                                                                                                                                                                                    

In [19]: from pathlib import Path                                                                                                                                                                                                                                                                                            

In [20]: from timeit import default_timer as timer                                                                                                                                                                                                                                                                           

In [21]: def load_all_ckans(identifiers): 
    ...:     start = timer() 
    ...:     for identifier in identifiers: 
    ...:         ckans = (Ckan(p) for p in identifier.glob('**/*.ckan')) 
    ...:         highest = max(ckans, default=None, key=lambda ck: ck.version) 
    ...:         highest.identifier if highest else None 
    ...:     end = timer() 
    ...:     print(end - start) 
    ...:                                                                                                                                                                                                                                                                                                                     

In [22]: load_all_ckans(Path('../CKAN-meta/').iterdir())                                                                                                                                                                                                                                                                     
1.4594180890126154

In [23]: load_all_ckans(Path('../CKAN-meta/').iterdir())                                                                                                                                                                                                                                                                     
1.4571303109405562
```
And that's all of them, not just the ones we'll be scheduling.  So I think we'll be fine and dandy for a synchronous scheduled task.